### DISCUSSION TITLE
Is there  any way to pass `env` or `executeCtx` from cloudflare workers to feTS server router?

### DISCUSSION THREAD

**Discussion by ryoppippi** [ardatan/feTS#375] [Q&A] [answered]
Hi
I'm wondering how to use `env` or `executeCtx` from cloudflare workers to `handler` functions in server router implementation.

**Comment by ardatan** [comment, marked as answer]
You can find both in the context as a second parameter;
```ts
handler(req, ctx) {
```

**Comment by ryoppippi** [comment]
Thanks!
